app/code/Magento/Customer/ViewModel/Address.php

```diff
@@ -0,0 +1,125 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Customer\ViewModel;
+
+use Magento\Directory\Helper\Data as DataHelper;
+use Magento\Customer\Helper\Address as AddressHelper;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Framework\View\Element\Block\ArgumentInterface;
+
+/**
+ * Customer address view model.
+ */
+
+ /**
+  * Address view model
+  */
+class Address implements ArgumentInterface
+{
+    /**
+     * Data helper
+     *
+     * @var DataHelper
+     */
+    private $helperData;
+
+    /**
+     * Address helper
+     *
+     * @var AddressHelper
```

Comment:
/**
     * @var AddressHelper $helperAddress
     */
    private $helperAddress;

Reply:
For the class properties you dont need to duplicate variable name.
@var DataHelper is correct. Or there are some Guidelines about that?